velox/exec/HashTable.cpp

```diff
@@ -621,6 +621,9 @@ void HashTable<ignoreNullKeys>::checkSize(int32_t numNew) {
     // hashing.
     auto newSize = std::max(
         (uint64_t)2048, bits::nextPowerOfTwo(numNew * 2 + numDistinct_));
```

Comment:
nit: can we put this logic into a method newTableSize(int32_t newEntries)?

Reply:
It's just only here so probably fine.  We can do that when we copy it to another place